tests/integration/common/data/interfaces/test_collections.py

```diff
@@ -311,3 +313,16 @@ def test_move_question_up_down(db_session, factories):
     assert q1.order == 2
     assert q2.order == 0
     assert q3.order == 1
+
+
+def test_update_collection_data(db_session, factories):
+    form = factories.form.build()
+    question = factories.question.build(form=form)
+    collection = factories.collection.build(collection_schema=form.section.collection_schema)
+
+    assert collection.data.get(str(question.id)) is None
```

Comment:
```suggestion
    assert str(question.id) not in collection.data
```

Unsure if intentional, but using `.get()` will return `None` if the key doesn't exist, so this is a bit of a weaker assertion than it probably needs to be?

Same for everywhere else we use `.get` - probably just do straight lookup with `[]`?